Cargo.toml

```diff
@@ -20,6 +20,10 @@ tinyvec = { version = "1.6.0", features = ["alloc"] }
 rsor = "0.1.3"
 duplicate = "0.4.1"
 dyn-clone = "1.0.9"
+crossbeam = "0.8"
+midi-msg = "0.4.0"
+midir = "0.8.0"
+read_input = "0.8.6"
```

Comment:
You should move these to `[dev-dependencies]`, since they are only used in the example.

Reply:
Thank you for the dev-dependencies suggestion. I'll address the other issue in the next conversation.